### PR TITLE
[librealsense2] new port

### PR DESCRIPTION
**Pull request by twMr7**
Intel RealSense SDK 2.0 library

### CODE REVIEW

File: ports/librealsense2/portfile.cmake [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+vcpkg_configure_cmake(
+    SOURCE_PATH ${SOURCE_PATH}
+    OPTIONS
+        -DENFORCE_METADATA=on
+        -DBUILD_EXAMPLES=off
+        -DBUILD_GRAPHICAL_EXAMPLES=off
+        -DBUILD_PYTHON_BINDINGS=off
+        -DBUILD_UNIT_TESTS=off
+        -DBUILD_WITH_OPENMP=off
```
Comment (Mixaill) [inline ports/librealsense2/portfile.cmake]: I think that OpenMP should not be disabled. This is a compiler feature, and it does not add any additional dependencies.

**Comment by twMr7** [comment]
It causes the [high CPU utilization issue](https://github.com/IntelRealSense/librealsense/issues/744) if library is compiled with OpenMP enabled. The issue is marked closed because it can be solved by disabling the OpenMP configuration. I assumed most of the developers would like to reserve enough computing power for real applications, not just streaming video frames and eat up 100% CPU utilization. I would recommend to keep the `BUILD_WITH_OPENMP=off` until someone got the streaming code patched.

**Comment by UnaNancyOwen** [comment]
<code>find_package(realsense2)</code> can not find librealsense2, because mismatch the configuration filename (<code>realsense2Config.cmake</code>) and directory name (<code>share/librealsense2</code>).
I will propose to change the package name to <code>realsense2</code> from <code>librealsense2</code>.

@ras0219-msft Do you have any good ideas to resolve it?

**Comment by twMr7** [comment]
I actually named this new port `realsense2` at first. I would love to make a new commit to rename it if the name `librealsense2` causes the trouble to get found.

**Comment by UnaNancyOwen** [comment]
@twMr7 OK, I confirmed that it works correctly. Thanks,

**Comment by ras0219-msft** [comment]
Thanks for the PR!

**Pull request merged**